jcg_testadapter_commons/src/main/scala/JSTestAdapter.scala

```diff
@@ -1,4 +1,5 @@
 
 trait JSTestAdapter extends TestAdapter {
+    def serializeAllCGs(inputDirPath: String, outputDirPath: String): Unit
```

Comment:
Why would this be JS specific?

Reply:
It's not, but I plan to remove it when I add timeouts, which will be handled outside of the adapters. That's going to be a different PR though!